gl_rubocop.gemspec

```
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative 'lib/gl_rubocop/version'
+
+NON_GEM_FILES = ['Gemfile', 'Gemfile.lock', 'Guardfile', 'bin/lint'].freeze
+
+Gem::Specification.new do |spec|
+  spec.name = 'gl_rubocop'
+  spec.version = GLRubocop::VERSION
+  spec.authors = ['Give Lively']
+
+  spec.summary = "A shareable configuration of Give Lively's rubocop rules."
+  spec.homepage = 'https://github.com/givelively/gl_rubocop'
+  spec.license = 'MIT'
+  spec.required_ruby_version = '3.1.6'
```

```suggestion
  spec.required_ruby_version = '>= 3.1'
```

DEFINITELY don't set a hard ruby version requirement in a gem.